src/table.js

```diff
@@ -1383,6 +1383,99 @@ Table.VIEWS = {
   full: 4,
 };
 
+/**
+ * Generates Consistency-Token and check consistency for generated token
+ * In-case consistency check returns false, retrial is done in interval
+ * of 5 seconds till 10 minutes, after that it returns false.
+ *
+ * @param {function(?error, ?boolean)} callback The callback function.
+ * @param {?Error} callback.err An error returned while making this request.
+ * @param {?Boolean} callback.resp Boolean value.
+ */
+Table.prototype.waitForReplication = function(callback) {
```

Comment:
This will need to be changed to use the ES6 class syntax for methods

Reply:
Done https://github.com/googleapis/nodejs-bigtable/pull/82/commits/5ba4e36e0e7b47a136f301cf7d9a01473e68add6